soyjs/exec.go

```diff
@@ -293,7 +292,7 @@ func (s *state) visitPrint(node *ast.PrintNode) {
 	s.indent()
 	s.js(s.bufferName, " += ")
 	for _, dir := range directives {
-		s.js("soy.$$", dir.Name, "(")
+		s.js(PrintDirectives[dir.Name].Name, "(")
```

Comment:
Heh I thought that too at first.  Is it worth it to rename `directives` to `directiveNodes` or something to make it clearer that it is the template objects and not a collection of `PrintDirectives`?